### PR TITLE
Quests for both apps

### PR DESCRIPTION
**Pull request by valentinludu**

### CODE REVIEW

**Review by Devorein — CHANGES_REQUESTED** [review]
Great everything seems to be working like before. A few small design changes are required.

image
1. The typography color seems to be using primary color rather than white
2. The width of the Quests section should be narrower according to the mockup
3. Clicking on quests when logged out should show a different modal like claim page
4. Scrollbar even when its not required

File: apps/scoutgametelegram/lib/session/getSession.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { getSession as getSessionBase } from '@packages/scoutgame/session/getSession';
 
 export const getSession = () => {
-  return getSessionBase({ sameSite: 'none' });
+  return getSessionBase({ sameSite: 'none', secure: true, domain: 'ngrok.app' });
```
Comment (Devorein) [inline apps/scoutgametelegram/lib/session/getSession.ts:4]: oops a leftover from dev

Comment (valentinludu) [reply apps/scoutgametelegram/lib/session/getSession.ts:4]: Reverted!

**Review by mattcasey — APPROVED** [review]
nice!

**Review by Devorein — APPROVED** [review]

**Pull request merged**